rmklaus12/lib/toy_robot/robot.rb

```diff
@@ -0,0 +1,77 @@
+module ToyRobot
+  class Robot
+    DIRECTIONS = %w[NORTH EAST SOUTH WEST]
+    attr_reader :x, :y, :facing
+
+    def initialize(x = 0, y = 0, facing = 'NORTH')
+      @x = x
+      @y = y
+      @facing = facing
+    end
+
+    def move_east
+      @x += 1
+    end
```

Comment:
I'm not a big fan of these move_x methods being public for the Robot class - I can see why they are here for testing purposes, but they do make it possible to move the robot into invalid positions if someone was to go and called them (when working in a team this is a very real concern).

You can keep the code as it is, and use this as a talking point in an interview - the key is to understand that this is something that you could improve upon. 

Suggested Solution: "black box test" the move method to ensure the functionality remains the same, make these methods private, and remove the tests that directly test these methods.

^^ it's important that you understand what I'm talking about here, as it could be a likely conversation in an interview situation.